dGame/dGameMessages/GameMessages.h

```diff
@@ -10,6 +10,8 @@
 #include "eEndBehavior.h"
 #include "eCyclingMode.h"
 #include "eLootSourceType.h"
+#include "eHelpType.h"
+#include "ePetAbilityType.h"
```

Comment:
should be able to replaced with forward decl

Reply:
👍